libs/perun/components/src/lib/authorized-groups-cell/authorized-groups-cell.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="mt-1">
+  <div
+    class="mb-1"
+    *ngFor="let group of groups.slice(0, itemsShown)"
+    [routerLink]="disableRouting ? null : ['/organizations', group.voId, 'groups', group.id]"
+    [perunWebAppsMiddleClickRouterLink]="disableRouting ? null : ['/organizations', group.voId.toString(), 'groups', group.id.toString()]"
+    [class.cursor-pointer]="!disableRouting">
+    {{authzVoNames.get(group.voId) + " : " + group.name}}
```

Comment:
I would like something like that for that link on hover on the chosen group: (we can discus it on daily)
```suggestion
    <span class="active-link">{{authzVoNames.get(group.voId) + " : " + group.name}}</span>
```